src/util/configure.ml

```diff
@@ -22,26 +22,26 @@ let args = [
 
 let main () =
   Arg.parse args ignore "enable lwt.unix and camlp4 features\noptions are:";
-  let f = open_out "src/unix/lwt_config" in
-  let print name var =
-    match var with
-    | None -> ()
-    | Some var -> Printf.fprintf f "%s: %b\n" name var
-  in
-  print"use_libev" !use_libev;
-  print"use_pthread" !use_pthread;
-  print"android_target" !android_target;
-  print"libev_default" !libev_default;
-  close_out f;
-  (* '-use-camlp4 false' (or none) will write a jbuild-ignore file directing jbuilder to ignore
-   *                     the camlp4 directory.
-   * '-use-camlp4 true' will write an empty file which does nothing
-   *
-   * This is a workaround required to overcome some weird camlp4 packaging behaviour
-   * where ocamlfind sometimes installs a dummy META file for it even though it's
-   * not actually installed. *)
-  let f = open_out "src/jbuild-ignore" in
-  (if !use_camlp4 = Some(true) then () else Printf.fprintf f "camlp4");
-  close_out f
+    let f = open_out "src/unix/lwt_config" in
```

Comment:
Looks like ocp-indent got this wrong. I wonder why? I don't think this should be indented more than the previous line (am I missing something?).